src/CoreBundle/Resources/public/javascript/table.js

```diff
@@ -107,6 +107,26 @@ function ChangeListMarking(fieldValue, formName) {
     );
 }
 
+function restoreFieldValueVersion(id) {
+    const valueColumnElement = document.querySelector(`table tr[data-record-id='${ id }'] *[data-field-restorable-value]`);
+    if (!valueColumnElement) {
+        return;
+    }
+
+    const encodedRestorableValue = valueColumnElement.dataset["fieldRestorableValue"];
+    
+    try {
+        const restorableValue = JSON.parse(encodedRestorableValue).value
```

Comment:
Has since been removed from `table.js` and moved to a dedicated file `versionHistoryActions.js` so it can become part of a bundling process in the future. With IE11 support, there is now a compatibility version loaded instead of this one — that file includes semicolons.